### PR TITLE
Ao/seizure classification

### PR DESCRIPTION
**Pull request by ambroiseodt**
## Checklist:
- [x] Create classifier and prediction layers --> heads.py
- [x] Update model to predict number of spikes --> spike_counting.py
- [x]  Create model to detect when a spike occurs in a given time window --> spike_detection.py
- [x] Update training for both models --> add scheduler to reduce learning rate when val_loss doesn't improve
- [x] Update evaluation for both model s--> add accuracy, specificity, recall, precision, F1_score metrics
- [x] Create custom Cross-entropy loss with cost-sensitive regularization --> custom_losses.py
- [x] Apply warmup steps for learning_rate --> learning_rate_warmup.py
- [x] Apply early stopping --> early_stopping.py
- [x] Check PEP8 conventions
- [x] Update example of config files for data, training and hyperparameters tuning
- [x] Find a way to localise spikes in time --> spike_detection.py
- [x] Create main.py file to train and evaluate models from terminal
- [ ] Update README.md to add model pipeline, command examples to run to train and test the model
- [x] Adapt the code in the case of multi-subject spike detection
- [x] Implement Nxk-fold cross-validation 
- [ ] Update spike_counting.py
- [ ] tweak spike_detection.py parameters and improve code

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 from loguru import logger
 
-def get_class_distribution(label, display, save, title):
+def get_class_distribution(label, display, save = False, title = None):
```
Comment (jcohenadad) [inline utils.py]:
```suggestion
def get_class_distribution(label, display, save=False, title=None):
```
PEP8

Comment (ambroiseodt) [reply utils.py]: Thanks, I will review the entire code to match to PEP8
